packages/docs/docs/contracts-api/verifiers/epoch-key-lite-verifier-helper.md

```diff
@@ -9,7 +9,8 @@ import { deployVerifierHelper } from '@unirep/contracts/deploy'
 import { defaultProver } from '@unirep/circuits/provers/defaultProver'
 import { Circuit } from '@unirep/circuits'
 
-let epochKeyLiteVerifierHelper = await deployVerifierHelper(accounts[0], Circuit.epochKeyLite) // deploys epoch key lite verifier helper contracts
+// deploys epoch key lite verifier helper contract
+let epochKeyLiteVerifierHelper = await deployVerifierHelper(accounts[0], Circuit.epochKeyLite)
```

Comment:
These should be `const`s to indicate that the variable is not being re-assigned.